Runtime/Player/ClientSimPlayerController.cs

```diff
@@ -200,12 +200,18 @@ public bool IsGrounded()
             return _characterController.isGrounded;
         }
 
-        private void Respawn()
+        public void Respawn()
         {
             Teleport(_sceneManager.GetSpawnPoint(false), false);
             _eventDispatcher.SendEvent(new ClientSimOnPlayerRespawnEvent { player = _playerApi.Player });
         }
 
+        public void Respawn(int index)
+        {
+            Teleport(_sceneManager.GetSpawnPoint(index), false);
```

Comment:
How does VRChat handle different indices? I assume it will also throw an error and crash Udon? Might be worth adding a note of the comparison or that this method is allowed to crash.